tasks/chain/economics.go

```diff
@@ -0,0 +1,151 @@
+package chain
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-pg/pg/v10"
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/raulk/clock"
+	"go.opencensus.io/tag"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/label"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/sentinel-visor/lens"
+	"github.com/filecoin-project/sentinel-visor/metrics"
+	chainmodel "github.com/filecoin-project/sentinel-visor/model/chain"
+	"github.com/filecoin-project/sentinel-visor/model/visor"
+	"github.com/filecoin-project/sentinel-visor/storage"
+	"github.com/filecoin-project/sentinel-visor/wait"
+)
+
+const (
+	idleSleepInterval = 60 * time.Second       // time to wait if the processor runs out of blocks to process
+	batchInterval     = 100 * time.Millisecond // time to wait between batches
+)
+
+var log = logging.Logger("message")
```

Comment:
name the logger "ChainEconomics"?

Reply:
Renamed to "chain" to match the package name